recipe-server/client/control/components/MultiPicker.js

```diff
@@ -0,0 +1,234 @@
+import React, { PropTypes as pt } from 'react';
+
+export default class MultiPicker extends React.Component {
+  static propTypes = {
+    unit: pt.string.isRequired,
+    plural: pt.string.isRequired,
+    options: pt.array,
+    // from redux-form
+    value: pt.oneOfType([pt.array, pt.string]),
+    onChange: pt.func,
+  };
+
+  static getActiveSelectOptions({ options }) {
+    if (!options) {
```

Comment:
Technically, it's not intended to ever be called without `options`, so you're correct that it's unnecessary. I've removed the explicit check and set a default `options` param, e.g. `..({ options = [] }) {...`